apps/notifications/src/panel/boot/stylesheets/actions.scss

```diff
@@ -60,7 +60,7 @@
 	}
 }
 
-.wpnc__main .wpnc__action-link {
+.wpnc__main button.wpnc__action-link {
```

Comment:
Increasing style specificity since we've added a :not pseudo-selector to the .wpnc__main button styles.

Same is done in few other places where needed.